tsconfig.json

```diff
@@ -4,7 +4,7 @@
     // "incremental": true,                   /* Enable incremental compilation */
     "target": "es6" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
     "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
-    "lib": ["es2015"] /* Specify library files to be included in the compilation. */,
+    "lib": ["es2015", "dom"] /* Specify library files to be included in the compilation. */,
```

Comment:
Why do we need to add dom libraries?

Reply:
I'm actually not entirely sure here -- when trying to build the project, I was running into an error with Axios: `Cannot find name 'AbortSignal'.`. Importing `dom` fixed this. Is this perhaps a versioning error or something else?

Reply:
Weirdly enough it seems to be working now without it, not sure what that was, but I've gone ahead and removed it for now.